src/microbe_stage/MicrobeHUD.cs

```diff
@@ -699,7 +699,7 @@ private void UpdateHoverInfo(float delta)
         }
 
         // Show the species name of hovered cells
-        foreach (var entry in stage.MicrobesAtMouse)
+        foreach (var entry in stage.HoverInfo.HoveredMicrobes)
```

Comment:
As we discussed loop variable name already, perhaps it would be worth to use the opportunity to rename the var away from `entry`, wouldn't it?